tests/Feature/Social/MastodonControllerTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\Feature\Social;
+
+use App\Http\Controllers\Backend\Social\MastodonController;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Validator;
+use Tests\TestCase;
+
+class MastodonControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * @dataProvider providerTestFormatDomain
+     */
+    public function testFormatDomain($case, $expected) {
+        $formatted = MastodonController::formatDomain($case);
+        self::assertEquals($expected, $formatted);
+
+        $validated = Validator::make(["domain" => $formatted], ["domain" => ["active_url"]]);
+
+        self:self::assertFalse($validated->fails());
+    }
+
+    public function providerTestFormatDomain(): array {
+        return [
+            ['https://uelfte.club', 'https://uelfte.club'],
+            ['http://uelfte.club', 'https://uelfte.club'],
+            ['uelfte.club', 'https://uelfte.club'],
+            ['great_username@uelfte.club', 'https://uelfte.club'],
+            ['@great_username@uelfte.club', 'https://uelfte.club'],
+            ['https://mastodon.sergal.org', 'https://mastodon.sergal.org'] # see issue 1182
+        ];
+    }
```

Comment:
Where is this called?

Reply:
I can run the first function manually and this is working for me.
But this format seems to be ignored by laravels test mechanism.

`php artisan test` doesn't show the test running.

Reply:
nvm. my fault.

![image](https://user-images.githubusercontent.com/4103693/201965776-fa57dbca-9fb7-4c04-9b04-61563cdc03fa.png)